Microsoft/PCI_Compliance_for_Windows_JSON/README.md

```diff
@@ -0,0 +1,18 @@
+# PCI Compliance for Windows JSON
+
+This is an updated version of the PCI Compliance for Windows app. It is updated to work with Windows Event Logs in the JSON formatted syntax. It is also utilizing the new Sumo Logic dashboard framework. 
+
+## Setup
+
+### Setup Collection
+
+This app only requires a standard Windows Event source(s) configured for the JSON-formatted event logs.
+
+### Modify Dashboard JSON
+
+Prior to importing the app, find and replace all instances of _\_sourceCategory=windows/events_ with your appropriate metadata for your Windows Event logs.
+
+
+### Import App
+
+Import your modified version of the JSON file into Sumo Logic.
```

Comment:
Add a link to help docs for how to import the JSON.

Reply:
Might also be good to link to the existing help doc about these dashboards to describe what they are telling you.